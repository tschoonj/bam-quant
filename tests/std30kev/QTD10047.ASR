File: C:\xrfdaten\2014KW30\chris\std30kev\std10047.spe
Start: 22.07.2014 10:18:12
Stop: 22.07.2014 10:19:52
BESSY: 298.8980595 298.5949384
IONI: 1.961027E-11 1.947562E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 66746
Run: 100000
Dead: 33
ICR: 113650
OCR: 0
Accumulated BESSY: 59690.4193014
Accumulated Ioni: 3.91352096E-09
N Accumulation: 200
Normfactor:  7.6566199e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ag
exp_x 132.720397949219 132.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  47   1     22.103   5993438.      2448.      78.25
