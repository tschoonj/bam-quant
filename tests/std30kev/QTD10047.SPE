$DATA:
0 2000
15276
10490
5105
1898
586
165
58
27
11
7
3
4
6
2
8
27
220
2801
4417
4371
4248
4111
3903
3828
3818
3690
3636
3522
3552
3355
3269
3136
3166
3125
2986
3004
2828
2915
2818
2737
2723
2667
2643
2639
2621
2667
2599
2495
2546
2662
2580
2660
2581
2606
2659
2637
2676
2754
2665
2772
2740
2792
2742
2902
2757
2853
2848
2849
2862
2964
2909
3016
2991
3048
3007
3020
3001
2823
2811
2846
2763
2656
2685
2700
2616
2476
2428
2390
2218
2303
2267
2148
2058
2062
2024
1960
1865
1825
1831
1831
1824
1865
1824
1674
1694
1703
1533
1486
1435
1352
1341
1226
1238
1202
1101
1080
1094
1052
1004
953
980
941
910
898
847
850
799
736
774
791
772
798
743
693
703
759
666
676
715
654
692
623
621
623
615
621
595
662
626
584
542
556
564
564
519
529
533
500
499
496
540
500
468
493
456
495
496
471
498
479
495
499
457
452
456
433
468
430
477
425
480
440
437
441
424
422
406
435
426
418
413
383
413
396
425
426
417
381
400
401
390
386
347
389
403
387
402
361
375
372
366
384
383
344
379
353
375
339
362
373
359
367
322
366
351
363
337
361
374
333
357
339
335
362
357
370
361
349
328
322
334
330
352
346
331
339
336
346
342
324
326
318
308
340
330
329
336
345
301
289
338
346
364
353
338
388
391
432
408
405
398
370
385
323
346
281
304
336
294
309
284
307
294
309
288
297
295
268
300
305
341
347
298
344
304
332
301
334
268
320
303
276
269
280
316
333
304
325
308
312
341
332
415
472
583
669
713
723
726
684
569
505
409
398
352
276
269
283
270
296
280
284
275
273
301
271
303
264
297
250
309
289
271
270
304
323
322
313
336
358
338
344
339
317
308
285
303
295
272
293
261
294
307
253
259
295
290
272
305
325
306
322
332
295
273
268
269
270
286
259
289
273
249
269
255
284
281
255
271
281
255
260
262
256
271
265
269
257
285
322
273
265
275
272
271
236
270
258
274
286
264
260
245
257
261
249
247
259
248
248
231
248
266
262
246
308
265
287
286
245
230
264
298
264
276
218
233
264
263
247
284
267
250
254
241
226
253
254
236
264
273
240
247
277
295
300
319
270
316
309
292
284
263
262
268
241
235
272
268
266
258
223
272
239
223
264
234
280
251
204
246
242
240
250
252
241
213
252
234
263
222
232
238
256
225
260
235
206
234
226
226
244
252
237
236
248
244
246
223
234
256
241
239
217
251
225
270
212
234
236
251
245
294
266
232
270
273
258
246
262
247
267
275
236
255
235
245
219
240
232
252
221
245
257
224
255
219
225
245
244
219
222
223
248
244
233
193
206
252
207
237
245
229
211
237
214
222
248
219
227
216
208
239
258
231
216
239
205
227
238
243
259
205
273
249
237
221
223
249
245
247
219
264
251
242
238
215
232
201
240
226
247
226
234
220
240
230
267
243
233
207
246
199
235
220
239
247
240
230
251
276
249
219
279
278
273
292
267
237
256
265
254
228
247
257
246
227
258
250
262
262
263
306
288
266
234
271
293
250
249
257
241
248
232
254
238
241
229
243
243
268
230
246
249
238
209
240
228
225
240
204
224
235
246
232
226
257
226
226
230
252
256
248
260
247
240
293
277
230
247
250
252
258
244
280
254
271
241
234
243
274
229
259
267
256
231
255
239
249
258
251
223
255
261
216
227
301
257
281
265
274
265
273
272
277
247
270
288
270
282
295
277
275
276
319
321
319
306
318
297
311
310
330
347
329
346
323
334
351
321
341
320
334
315
305
355
347
348
336
351
369
354
392
423
454
553
671
863
1211
1729
2546
3332
4779
6143
8036
9893
11700
13494
14741
15516
15689
15398
14519
12985
11200
9237
7166
5583
4077
2928
1956
1341
982
693
579
418
350
301
334
296
282
296
338
321
300
266
315
303
283
288
289
296
331
282
286
298
300
289
327
324
287
282
313
281
325
313
302
290
287
297
313
323
315
319
294
289
326
307
305
310
317
348
327
315
332
335
330
324
328
340
349
365
364
340
371
376
364
374
403
378
429
428
437
486
552
720
852
1043
1304
1556
1827
2140
2468
2657
2681
2708
2711
2439
2146
1939
1664
1427
1096
988
871
789
744
709
755
714
663
723
653
627
554
538
549
490
445
423
385
421
388
377
390
412
389
348
411
406
368
412
405
387
374
398
420
367
378
395
391
418
385
417
404
374
397
402
409
400
436
419
414
414
385
406
416
356
434
427
412
411
426
433
476
483
464
489
470
506
507
487
514
489
516
505
535
514
517
548
534
544
588
564
598
592
535
576
649
617
627
594
619
674
660
634
727
690
714
772
762
742
759
775
762
830
800
872
873
963
908
962
1006
1010
1049
1093
1116
1242
1246
1367
1479
1502
1560
1565
1584
1660
1682
1763
1842
1814
1904
1880
1944
2032
1949
1910
1980
1916
1846
1930
1949
1913
2042
1901
1894
1917
1949
1847
1952
1960
1858
1939
1945
1996
1839
1916
1899
1849
1877
1987
1941
1858
1885
1915
1936
1901
2023
1887
1958
2031
2035
1979
2064
2176
2125
2088
2199
2247
2333
2374
2382
2426
2571
2649
2630
2887
2903
3083
3260
3341
3756
3970
4681
5521
6703
8915
11899
16356
22695
30942
42850
56664
74694
94381
117594
141773
166985
191996
216552
240473
264106
285348
305214
322219
335935
344427
346793
339498
325833
303671
276033
240872
204162
168641
133548
102851
76265
55687
38936
26844
18263
12208
8212
5792
4293
3308
2610
2187
2044
1766
1650
1498
1412
1286
1288
1236
1172
1072
1015
1007
1014
1005
950
880
890
851
889
822
876
841
865
767
779
791
796
814
785
766
784
787
763
736
750
703
703
697
726
643
716
662
681
655
651
631
678
675
675
598
662
621
608
680
698
668
636
662
627
631
667
642
674
700
619
651
652
676
706
686
673
696
693
700
751
761
750
706
745
776
757
820
778
786
819
836
871
866
886
875
902
1022
929
1041
1002
1138
1201
1284
1438
1716
2055
2571
3562
4803
6545
8962
12279
16471
21577
27513
34416
41729
48462
55054
60670
64143
66184
66342
63556
59934
53732
47036
40571
33027
26427
20887
16074
12177
8972
6877
5441
4753
4494
4613
5196
6003
6855
7946
8852
10107
10923
11381
11509
11208
10595
9774
9032
7770
6518
5354
4157
3129
2407
1763
1198
870
570
390
288
207
165
118
97
76
85
63
61
70
55
53
56
51
59
39
51
41
57
29
34
37
64
31
44
34
45
46
43
45
40
41
35
43
45
53
27
36
40
33
34
34
34
37
20
23
31
32
32
36
28
18
23
28
24
28
14
27
33
18
23
27
27
23
24
26
17
22
28
21
15
10
16
24
22
16
19
15
12
20
18
16
30
25
16
14
28
25
12
15
18
17
20
15
13
18
21
11
8
13
13
15
14
9
12
13
13
16
12
12
15
14
14
10
17
8
13
11
9
14
12
14
11
15
10
17
15
17
17
12
18
11
16
18
9
15
16
13
16
15
10
15
14
6
8
9
6
18
7
8
13
11
9
10
11
10
8
6
6
10
13
15
4
8
8
12
7
10
10
10
8
12
9
10
13
16
11
13
10
6
7
11
10
11
10
11
10
8
4
10
10
10
14
4
11
10
8
10
6
11
7
10
11
4
7
8
3
11
10
7
9
7
8
10
12
5
3
6
6
9
4
9
9
14
7
6
10
5
5
11
10
9
6
7
7
3
6
13
5
10
11
9
9
8
9
5
9
10
8
7
4
5
7
14
9
5
7
9
10
5
7
11
5
4
10
8
7
16
12
10
5
10
8
12
10
10
10
10
13
18
15
9
15
20
10
10
9
6
11
9
10
11
13
8
7
11
10
11
10
13
8
7
11
7
7
2
11
12
11
9
4
11
8
8
9
13
10
9
15
8
16
7
6
11
10
9
8
13
10
8
17
8
11
12
8
13
10
19
11
9
9
11
13
7
16
11
14
8
5
14
14
7
15
7
11
8
11
10
17
15
11
19
19
12
5
9
6
16
8
13
7
18
11
18
11
14
14
10
14
13
17
14
17
11
14
6
13
12
13
6
6
12
7
8
15
16
10
10
13
10
11
13
9
15
9
8
11
8
9
14
14
10
12
14
25
16
14
15
18
23
14
9
15
15
10
22
10
14
16
16
13
15
16
12
13
9
7
13
15
17
15
14
11
19
12
8
11
16
19
12
18
24
23
23
18
25
24
28
28
26
26
26
33
19
30
21
26
18
33
21
18
17
18
18
18
14
16
23
12
13
18
17
17
15
14
13
14
15
12
6
10
6
11
8
3
6
6
3
6
10
2
9
11
8
6
8
8
9
6
5
6
6
6
3
7
2
7
5
9
5
5
6
6
6
4
4
13
7
12
8
4
10
8
4
4
6
9
2
7
6
11
1
9
8
2
6
10
5
6
9
11
6
9
5
8
8
8
11
8
9
9
5
5
9
5
7
5
12
8
10
12
15
12
17
14
16
30
25
35
44
57
52
68
68
91
84
98
102
107
122
153
157
163
147
147
173
160
155
147
154
140
126
107
103
91
97
95
73
62
53
57
38
26
18
17
8
9
12
12
14
10
12
5
5
6
4
8
8
10
4
8
5
4
7
5
4
7
8
9
9
9
5
10
4
5
1
4
5
10
5
4
6
5
9
6
9
11
5
11
9
9
8
6
5
9
6
10
7
10
10
10
17
13
10
12
17
12
17
15
24
20
19
23
28
29
24
32
36
32
31
25
32
29
25
28
22
25
27
21
22
27
24
17
16
17
13
12
10
15
16
11
9
8
12
13
9
14
10
12
14
5
15
24
12
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10047.spe
Start: 22.07.2014 10:18:12
Stop: 22.07.2014 10:19:52
BESSY: 298.8980595 298.5949384
IONI: 1.961027E-11 1.947562E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 66746
Run: 100000
Dead: 33
ICR: 113650
OCR: 0
Accumulated BESSY: 59690.4193014
Accumulated Ioni: 3.91352096E-09
N Accumulation: 200
Normfactor:  7.6566199e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ag
 
exp_x 132.720397949219 132.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  47   1     22.103   5993438.      2448.      78.25
