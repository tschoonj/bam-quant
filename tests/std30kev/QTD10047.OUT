     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:54  บ   
     บ  Spectrum: QTD10047.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =   311.5   last change =   .08%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -25.3 ๑   100.0   บ     -25.3 ๑      .8   บ   
     บ  GAIN (eV/ch)  บ     20.203 ๑   2.000   บ    20.203 ๑    .001   บ   
     บ det NOISE (eV) บ      132.0 ๑    40.0   บ     132.0 ๑     4.0   บ   
     บ  FANO factor   บ       .107 ๑    .050   บ      .107 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ag-Ka                           5993438. ๑   2448.           บ   
     บ    KA1        22.163    .65212     3908452. ๑   1597.    101.4  บ   
     บ             1098.263    261.06        1377.           8.98E-01  บ   
     บ    KA2        21.990    .34788     2084986. ๑    852.     82.3  บ   
     บ             1089.700    260.30        1419.           9.03E-01  บ   
     บ                                                                 บ   
     บ  2 Ag-Kb                           1104714. ๑   1052.           บ   
     บ    KB1        24.942    .56579      625039. ๑    596.     36.5  บ   
     บ             1235.816    272.97         881.           8.07E-01  บ   
     บ    KB3        24.912    .29143      321947. ๑    308.     36.7  บ   
     บ             1234.331    272.84         885.           8.08E-01  บ   
     บ    KB2        25.456    .14278      157729. ๑    153.     37.1  บ   
     บ             1261.258    275.11         780.           7.88E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
